code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -59,6 +59,10 @@
 
 	var/fire_resist = 1 //0 to 1; lower is better as it is a multiplier.
 
+	// *** Sunder *** //
+	var/sunder_recover = 0.1 // How much sunder is recovered per tick
```

Comment:
```suggestion
	var/sunder_recover = 3 // How much sunder is recovered per tick
```
A life tick happens in around 2 seconds each. Better increase this drastically.
With this value the xeno will need a full minute to regen from zero to full again.